$NetBSD: patch-libraries_liblmdb_Makefile,v 1.2 2016/02/16 12:16:28 mef Exp $

Use proper install commands.

 clean:
 	rm -rf $(PROGS) *.[ao] *.[ls]o *~ testdb
--- libraries/liblmdb/Makefile.orig	2016-02-06 06:22:15.000000000 +0900
+++ libraries/liblmdb/Makefile	2016-02-16 21:08:37.000000000 +0900
@@ -48,10 +48,10 @@ install: $(ILIBS) $(IPROGS) $(IHDRS)
 	mkdir -p $(DESTDIR)$(libdir)
 	mkdir -p $(DESTDIR)$(includedir)
 	mkdir -p $(DESTDIR)$(mandir)/man1
-	for f in $(IPROGS); do cp $$f $(DESTDIR)$(bindir); done
-	for f in $(ILIBS); do cp $$f $(DESTDIR)$(libdir); done
-	for f in $(IHDRS); do cp $$f $(DESTDIR)$(includedir); done
-	for f in $(IDOCS); do cp $$f $(DESTDIR)$(mandir)/man1; done
+	for f in $(IPROGS); do $(BSD_INSTALL_PROGRAM) $$f $(DESTDIR)$(prefix)/bin; done
+	for f in $(ILIBS); do $(BSD_INSTALL_LIB) $$f $(DESTDIR)$(prefix)/lib; done
+	for f in $(IHDRS); do $(BSD_INSTALL_DATA) $$f $(DESTDIR)$(prefix)/include; done
+	for f in $(IDOCS); do $(BSD_INSTALL_MAN) $$f $(DESTDIR)$(prefix)/$(PKGMANDIR)/man1; done
 
 clean:
 	rm -rf $(PROGS) *.[ao] *.[ls]o *~ testdb
